web/channels/room_channel.ex

```diff
@@ -1,33 +1,24 @@
 defmodule VerkWeb.RoomChannel do
   use Phoenix.Channel
+  alias VerkWeb.TrackingJobsHandler
 
-  @doc """
-  Authorize socket to subscribe and broadcast events on this channel & topic
-
-  Possible Return Values
-
-  `{:ok, socket}` to authorize subscription for channel for requested topic
-
-  `:ignore` to deny subscription/broadcast on this channel
-  for the requested topic
-  """
+  @doc false
   def join("rooms:jobs", _, socket) do
-    Process.flag(:trap_exit, true)
-    send self, :stats
-
+    :ok = GenEvent.add_mon_handler(Verk.EventManager, {TrackingJobsHandler, self}, self)
```

Comment:
Yes we still need Verk to be running but at least the only thing that will fail is the room channel code. So it should not break any application. And Verk Web is dependent on Verk anyway (for example to show running jobs, inspect a job etc)